### ISSUE TITLE
Small Dataset pulled when you call linear_training_data()

### ISSUE THREAD

**Issue by drachimera** [pymc-devs/pymc#4676]
This is a request from the DEVS on the discourse channel:

On the homepage there is this function:
X, y = linear_training_data()
when you copy and paste it just to see if you have a working install, it bombs. Perhaps we could put a very small test dataset in the install so a function like this works out of the box?

When I was installing and trying to see if the install worked, the example didn't work because there is no data when you call linear_training_data().  

This issue would resolve that so that if someone copies and pastes the first example they see, it works if they have a working install.

Full discussion is here: https://discourse.pymc.io/t/attributeerror-module-arviz-has-no-attribute-geweke/6818/27

**Comment by farhanreynaldo** [comment]
Hi, what if we create simple fake data for the `linear_training_data()`? I would suggest something like this:

```python
import numpy as np

np.random.seed(12)

def linear_training_data():
    X = np.random.normal(size=100)
    y = 1.2 * X + np.random.normal(size=100)
    return X, y
```

But I am not quite sure where to put this function. Or, we might as well change the homepage data snippet with existing PyMC3 examples, such as [milk data](https://github.com/pymc-devs/pymc-examples/blob/main/examples/data/milk.csv).

**Comment by michaelosthege** [comment]
closed by #4713
